labelCloud/definitions/label_format.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum
+from typing import List
+
+
+class LabelFormat(str, Enum):
+    VERTICES = "vertices"
+    CENTROID_REL = "centroid_rel"
+    CENTROID_ABS = "centroid_abs"
+    KITTI = "kitti"
+    KITTI_UNTRANSFORMED = "kitti_untransformed"
+
+    @classmethod
+    def list(cls) -> List["LabelFormat"]:
```

Comment:
It might be possible to use the `__contain__` built-in method of Enum https://docs.python.org/3/library/enum.html#enum.EnumType.__contains__